tasks/test-metadata/0.2/test-metadata.yaml

```diff
@@ -79,11 +79,10 @@ spec:
 
         COMPONENT_CONTAINER_IMAGE=$(jq -r --arg component_name "$KONFLUX_COMPONENT_NAME" '.components[] | select(.name == $component_name) | .containerImage' <<< "$SNAPSHOT")
 
-
-        # In case is coming from /retest event then it's a pull_request
-        # if event type is not push and pull_request_number is not empty then is a pull_request
-        if [ "$EVENT_TYPE" != "push" ] && [ -n "$PULL_REQUEST_NUMBER" ]; then
+        if [[ "$EVENT_TYPE" != "push" && -n "$PULL_REQUEST_NUMBER" ]]; then
           EVENT_TYPE="pull_request"
+        elif [[ -z "$PULL_REQUEST_NUMBER" && "$EVENT_TYPE" != "push" ]]; then
+          EVENT_TYPE="push"
```

Comment:
thinking if we could simplify it in a future:
```
        if [[ "$PULL_REQUEST_NUMBER" != ""]]; then
          EVENT_TYPE="pull_request"
        else
          EVENT_TYPE="push"
        fi
```
but for now let's keep it as is